src/renderer/wg_engine/tvgWgGeometry.h

```diff
@@ -373,7 +337,7 @@ struct WgVertexBufferInd {
         float cosine = dot(nrm, nrm1);
         float angle = std::acos(dot(nrm1, -nrm2));
         float miterRatio = 1.0f / (std::sin(angle) * 0.5f);
-        if (miterRatio <= miterLimit) {
+        if ((miterRatio <= miterLimit) && (!tvg::zero(cosine))) {
```

Comment:
the same problem may be with sin, maybe check the angle value? 
also this change should be done as a separate commit I think

Reply:
Make sence, fixed